llvm/lib/Target/DirectX/DXILMetadata.h

```diff
@@ -33,6 +33,7 @@ class ValidatorVersionMD {
 };
 
 void createShaderModelMD(Module &M);
+void createDXILVersionMD(Module &M);
```

Comment:
Consider expanding MD as Metadata in the function name `createDXILVersionMD` as `createDXILVersionMetadata`. Similarly, `createShaderModelMetadata`.